torch_xla/__init__.py

```diff
@@ -46,6 +46,8 @@ def _set_missing_flags(flags, sets):
 def _setup_xla_flags():
   flags = os.environ.get('XLA_FLAGS', '').split(' ')
   flags = _set_missing_flags(flags, (('xla_cpu_enable_fast_math', 'false'),))
+  flags = _set_missing_flags(
+      flags, (('xla_gpu_simplify_all_fp_conversions', 'false'),))
```

Comment:
Does this flag have a perf impact that we should consider?

Reply:
I don't know. It's made by Jack. Let me double check.

Reply:
By looking at https://github.com/tensorflow/tensorflow/commit/6a98a2c6f8e4074739b8ad888a5fbd32115e105e, we probably should have this flag set `true`. Let me try that.